### DISCUSSION TITLE
Fiducial Model of Scattering Time in Catalog 1

### DISCUSSION THREAD

**Discussion by InesPM** [chime-frb-open-data/community#30] [Q&A] [answered]
Hello, 

I'm trying to reproduce the fiducial model for the scattering time distribution of Catalog 1 using the equation for the lognormal distribution given in Appendix C.1 and the fiducial tau scale and shape given in Table 4.
However, I don't get the expected shape. I believe xi might have been forgotten from the divisor.

In this ipython code I reproduce the fiducial model as given in the paper (lognormal_fiducial) and the modified version I think might have been used instead (lognormal_modified):

```
def lognormal_fiducial(x, shape, scale):
    lognormal = 1 / (shape * (x / scale) * np.sqrt(2*pi)) \
                * np.exp(-(np.log(x / scale)) ** 2 / (2 * shape ** 2))
    return(lognormal)

def lognormal_modified(x, shape, scale):
    lognormal = 1 / (shape / scale * np.sqrt(2*pi)) \
                * np.exp(-(np.log(x / scale)) ** 2 / (2 * shape ** 2))
    return(lognormal)

tau = np.logspace(-1.1, 1, 100)

plt.plot(tau, lognormal_fiducial(tau, 1.72, 2.02), label='fiducial', color='C0')
plt.plot(tau, lognormal_modified(tau, 1.72, 2.02), label='modified', color='C1')
plt.xscale('log')
plt.xlim(5e-2, 200)
plt.ylim(0,1.8)
plt.xlabel('Scattering timescale (ms)')
plt.legend()
plt.locator_params(axis='y', nbins=8)
plt.show()
```

I get the following plot:

![Figure_1](https://user-images.githubusercontent.com/29770213/126155433-3bbcc562-2424-422d-8925-6396163fabf6.png)

The blue line (label='fiducial') uses the equation as given in the appendix. The orange line (label='modified') shows the modified version, and it seems to match the black line of Figure 17, right panel. Note that I use similar plot limits as in Fig. 17, but in ms.

I would like to know which function is correct, or if I'm misinterpreting something.

I'm using python 3.8 in a linux machine.

Thanks in advance.

**Comment by kiyo-masui** [comment, marked as answer]
Hi @InesPM. In the right panel of figure 17 note we are plotting `tau * lognormal_fiducial(tau)`, which I think might solve the discrepancy (although I could be wrong, I didn't comb through your post too carefully).

The reason we plot it in this way is because of the logarithmic binning in the horizontal axis. If you have a PDF P(x), then the PDF for ln(x) is x * P(x).

**Comment by kiyo-masui** [comment]
I'm closing the issue since I'm pretty sure my previous comment accounts for the missing factor. Let me know if you disagree and I'll reopen.

**Comment by InesPM** [comment]
Hi @kiyo-masui, thanks for your answer, that seems to solve the issue.
